xml/cap_depl_passwords.xml

```diff
@@ -0,0 +1,129 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter version="5.0" xml:id="cha.cap.manage-passwords"
+  xmlns="http://docbook.org/ns/docbook"
+  xmlns:xi="http://www.w3.org/2001/XInclude"
+  xmlns:xlink="http://www.w3.org/1999/xlink">
+ <info>
+  <title>Managing Passwords</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  The various components of &productname; authenticate to each using passwords
+  that are automatically managed by the &cap; secrets-generator. The only
+  passwords managed by the cluster administrator are passwords for human users,
+  and service passwords set in the cluster's <filename>values.yaml</filename>
+  configuration file (e.g. the example
+  <filename>scf-config-values.yaml</filename> used in this guide).
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    The cluster administrator password is initially defined in the deployment's
+    <filename>values.yaml</filename> file with
+    <literal>CLUSTER_ADMIN_PASSWORD</literal>
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    The Stratos Web UI provides a form for users, including the administrator,
+    to change their own passwords
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    User logins are created (and removed) with the Cloud Foundry Client, cf CLI
```

Comment:
I am not aware of `cf` commands for creating and managing users.
I believe that Pivotal does it through their web console.
I believe that it can be done through Stratos (our console, actually becoming the official CF web console)

Reply:
`cf` commands, e.g. `cf create-user`, for user management can be found in the User Admin section of `cf help -a`.

Reply:
Thank you for the correction. Confirmed. Retracting my change request.